frontend/javascripts/oxalis/view/context_menu.js

```diff
@@ -510,6 +515,19 @@ function NoNodeContextMenuOptions(props: NoNodeContextMenuProps) {
             Create new Tree here{" "}
             {!isVolumeBasedToolActive && !isBoundingBoxToolActive ? shortcutBuilder(["C"]) : null}
           </Menu.Item>,
+
+          <Menu.Item
+            className="node-context-menu-item"
+            key="load-agglomerate-skeleton"
+            disabled={!isAgglomerateMappingEnabled.value}
+            onClick={() => loadAgglomerateSkeletonAtPosition(globalPosition)}
+          >
+            {isAgglomerateMappingEnabled.value ? (
+              "Create Agglomerate Skeleton"
```

Comment:
@daniel-wer Do you think this wording is good? The other button (in the left sidebar) uses "Import Skeleton for Centered Segment" which is too unspecific, as the mapping context is missing in the context menu. So, I like the suggested wording more. Not sure whether "create" should be changed to "import" or "load"..

And another question for you, @daniel-wer: Do you agree that the "Import Skeleton for Centered Segment" button should be removed, since the new context menu entry covers this?

Reply:
> Do you think this wording is good? The other button (in the left sidebar) uses "Import Skeleton for Centered Segment" which is too unspecific, as the mapping context is missing in the context menu. So, I like the suggested wording more. Not sure whether "create" should be changed to "import" or "load"..

I'd go with `Import Agglomerate Skeleton` as that most closely resembles what's actually happening.

> And another question for you, @daniel-wer: Do you agree that the "Import Skeleton for Centered Segment" button should be removed, since the new context menu entry covers this?

Yes, I think it can be removed. However, do we have the possibility to tell users where the functionality moved to? Otherwise, I imagine, this could be a bit frustrating. Maybe, leave a disabled button with a tooltip explaining to use the context menu for a couple of weeks, then remove it?

Also, on another note, I didn't expect the shortcut to be removed. At least for me (and I know from at least one other "power user"), it's a use case to quickly load a bunch of agglomerate skeletons which would be quite a bit faster if a shortcut was available. Would it be easy to add that back and mention it in the context menu as is done for other shortcuts? This is not a deal-breaker for me and we could also merge this without the shortcut, but I would expect a discuss issue regarding that, rather quickly :see_no_evil:

Reply:
Sounds very reasonable to me :+1: @Dagobert42 could you then please do the following?

- rename menu item to "Import Agglomerate Skeleton"
- "leave a disabled button with a tooltip explaining to use the context menu" + open a follow-up issue to remove the button again in a couple of weeks
- re-add the shift+middleclick shortcut
- mention the shortcut in the context menu (for the middleclick you can probably reuse the `icon-statusbar-mouse-wheel.svg` icon. just search the codebase for its usage).

Let me know in case you have questions or get stuck somewhere! Thank you!